lotsqrl/game.py

```diff
@@ -10,11 +11,12 @@
 
 
 class Options(object):
-    def __init__(self):
+    def __init__(self, **options):
         self.automata_steps = 4
         self.graphical_tiles = True
         self.map_width = 25
         self.map_height = 25
+        self.graphics_base_dir = pathlib.Path(options.get("graphics_base_dir", "graphics"))
```

Comment:
What is the advantage of pathlib versus os.path.join?

Reply:
It make more readable code. For example:

````
def write_to_file(base_path, filename, text)
    my_file = base_path / filename
    my_file.write_text(text, encoding="utf-8")  # There is also a write_binary method

my_path = Path("base_dir")
write_to_file(my_path, "foo.txt", "bar")
````
Is more readable than:
````
def write_to_file(base_path, filename, text)
    with open(os.path.join(base_path, filename)) as f:
        f.write(text)

my_path = "base_dir"
write_to_file(my_path, "foo.txt", "bar")
````
It also has a lot of nice features, like 
 * Support for glob operations (recursively getting all the files that respect a certain pattern), no need for _glob_
 * A lot of methods/properties for dealing with paths

You can look in the official documentation for more reference: https://docs.python.org/3/library/pathlib.html